### ISSUE TITLE
Not skipped parallel commands after a command stage failed.

### ISSUE THREAD

**Issue by takyoshi** [walter-cd/walter#158]
I have a question about parallel commands after a command stage fails.
I expected that each parallel command would skip after other command stage failed, but these parallel commands run.
If there are some solution, sorry for my overlooking.

Below pipeline yaml that I used:
```yml
pipeline:
  - name: Start Deploy Sync
  - name: exit error
    type: command
    command: exit 1
  - name: will be skiped
    type: command
    command: echo "will be skipped"
  - name: start parallel
    parallel:
    - name: parallel 1 ( expect skip )
      type: command
      command: echo "expect skip"
    - name: parallel 2 ( expect skip )
      type: command
      command: echo "expect skip"
    - name: parallel 3 ( expect skip )
      type: command
      command: echo "expect skip"
  - name: will be skipped
    type: command
    command: echo "will be skipped"
  - name: End Deploy Sync
```

results:

![selection_040](https://cloud.githubusercontent.com/assets/13774847/15569222/fad3e77e-236a-11e6-9dcb-d2a928080581.png)

```
$ walter -v
Walter version "v1.3.0-57-g7409903"
```

**Comment by takahi-i** [comment]
Thank you very much for the valuable report and the detailed information for the reproduction @takyoshi!  I will look into the behavior.

**Comment by takahi-i** [comment]
The bug was fixed with #159. Hopefully the fix works for you. Let me know if you have further problems @takyoshi!
